### ISSUE TITLE
Feature request: Open Group in private Window

### ISSUE THREAD

**Issue by lviolal** [Morikko/sync-tab-groups#128]
I think the title says it all ;)

**Comment by Morikko** [comment]
This behavior is not supported because a private group will be lost once closed so it is always a temporary one.

But you can open a private window, and click on the bottom left button to add it as a group. But it won't be kept once the window will be closed.

**Comment by Morikko** [comment]
![image](https://user-images.githubusercontent.com/7831572/42135926-cdd8a392-7d52-11e8-8bf1-c1793bec1749.png)

**Comment by MaximilianKohler** [comment]
Let me know if I should make a new issue for this. 

I have been using sync tab groups for years, but just today the behavior of using them in new private windows changed and I don't see anything in the options that covers this. 

The only thing I can think of is that I had **firefox sync** active previously, and recently deactivated it. 

## Old behavior:

1. Have normal window open. Have 4 existing groups. 
2. Open new private window. 
3. Open existing group (not currently open/active) in the new private window. The blue "OPEN" shows next to the group. 
4. Open a new tab, or change the URL of existing tab. 
5. Close the private window. 
6. All changes are saved. Next time I open that group (whether in a private window or not), everything I changed is saved - Eg: if I created a new tab it's still there. 

## New behavior:

After step 5, no changes are saved.

**Comment by Morikko** [comment]
Sorry the project is no more maintained, you should try the suggested alternative. Else you can fork the project and fix it yourself but I guess a lot of things are getting old here with the evolving extension API.

https://github.com/Morikko/sync-tab-groups#the-project-is-no-more-maintained
